plugins/auth-backend/package.json

```
@@ -51,20 +53,25 @@
     "@types/passport": "^1.0.3",
     "compression": "^1.7.4",
     "connect-session-knex": "^3.0.1",
+    "cookie": "^0.5.0",
     "cookie-parser": "^1.4.5",
+    "cookie-signature": "^1.2.1",
     "cors": "^2.8.5",
+    "d": "^1.0.1",
+    "e": "^0.2.32",
```

do you import these anywhere?
```suggestion
    "d": "^1.0.1",
    "e": "^0.2.32",
```

---

Nope. removed from package.json

---

This used to be a bug in our ESLint rule, fixed for a while now though